src/common/components/clusterWizard/networkingSteps/ManagedNetworkingControlGroup.tsx

```diff
@@ -7,19 +7,28 @@ import { useTranslation } from '../../../hooks/use-translation-wrapper';
 export interface ManagedNetworkingControlGroupProps {
   disabled: boolean;
   tooltip?: string;
+  tooltipUmnDisabled?: string;
 }
 
 const GROUP_NAME = 'managedNetworkingType';
 export const ManagedNetworkingControlGroup = ({
   disabled = false,
   tooltip,
+  tooltipUmnDisabled,
 }: ManagedNetworkingControlGroupProps) => {
   const tooltipProps: TooltipProps = {
     hidden: !tooltip || !disabled,
     content: tooltip,
     position: 'top',
   };
+
   const { t } = useTranslation();
+
+  const tooltipPropsUmnDisabled: TooltipProps = {
```

Comment:
IMO If the entire control is disabled, we should show the default `tooltipProps`, no the specific one about UMN.

```
const showUmnTooltip = !isDisabled && !!tooltipUmnDisabled
```

Reply:
No need for this anymore if we only have one `disabled` and one `tooltip` prop